docs/contributing/standards.md

```diff
@@ -0,0 +1,69 @@
+---
+id: standards
+title: Contributing
+sidebar_label: Standards
+---
+
+# Standards
+
+The following standards are used by FAST-DNA contributors for consistency across this repository.
+
+## Core principles
+
+* Build for maintainability
+* Build for scalability
+* Build for performance
```

Comment:
Per the other pull request comment do you want to add accessibility here

Reply:
yes.  and core principals were removed.